File: VB_Example01/VB_Example01/Form1.vb
Public Class Form1
    Private Sub Button1_Click(sender As Object, e As EventArgs) Handles Button1.Click
        If Me.RichTextBox1.BackColor = Color.Orange Then
            Me.RichTextBox1.BackColor = Color.Red
        Else
            Me.RichTextBox1.BackColor = Color.Orange
        End If
    End Sub

    Private Sub Button2_Click(sender As Object, e As EventArgs) Handles Button2.Click
        If Me.RichTextBox1.ForeColor = Color.Green Then
            Me.RichTextBox1.ForeColor = Color.Blue
        Else
            Me.RichTextBox1.ForeColor = Color.Green
        End If
    End Sub

    Private Sub Button3_Click(sender As Object, e As EventArgs) Handles Button3.Click
        Me.RichTextBox1.ForeColor = Color.Black
        Me.RichTextBox1.BackColor = Color.White
    End Sub
End Class

File: CS_Example01/CS_Example01/Form1.cs
namespace CS_Example01
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {
            if (this.richTextBox1.BackColor == Color.Orange) {
                this.richTextBox1.BackColor = Color.Red;
            } else {
                this.richTextBox1.BackColor = Color.Orange;
            }
        }

        private void button2_Click(object sender, EventArgs e)
        {
            if (this.richTextBox1.ForeColor == Color.Green) {
                this.richTextBox1.ForeColor = Color.Blue;
            } else {
                this.richTextBox1.ForeColor = Color.Green;
            }
        }

        private void button3_Click(object sender, EventArgs e)
        {
            this.richTextBox1.ForeColor = Color.Black;
            this.richTextBox1.BackColor = Color.White;
        }
    }
}